crypto/evp/evp_enc.c

```diff
@@ -276,6 +276,7 @@ int EVP_CipherInit_ex(EVP_CIPHER_CTX *ctx, const EVP_CIPHER *cipher,
         case NID_rc5_ecb:
         case NID_rc5_cfb64:
         case NID_rc5_ofb64:
+        case NID_rc2_ecb:
```

Comment:
At some point, we should stop maintaining this absurd list, and instead simply try to fetch, and fall back to legacy when that fails.

Reply:
At some point there should be no legacy (so also no list ) :)

Reply:
Yes, absolutely...  but I'm thinking that *even now*, we could throw away that list.  It was a help before we got everything down, but at this point, I think the new `EVP_CIPHER` and `EVP_MD` have matured enough to make that change.